tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932380.yaml

```
@@ -166,27 +166,11 @@ tests:
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
               Accept-Encoding: gzip, deflate, br
               Accept-Language: en-us,en;q=0.5
-            method: GET
+            method: POST
             port: 80
-            uri: "/?foo=time+warner+"
+            uri: "/post?"
             version: HTTP/1.1
+            data: |
+              payload=payload=While this is a challenging time for us all, we are busy helping customers manage playout infrastructure in ways that were just dreams only a couple of years ago.
```

I think this may be a typo:

```suggestion
              payload=While this is a challenging time for us all, we are busy helping customers manage playout infrastructure in ways that were just dreams only a couple of years ago.
```